src/NHibernate.Test/NHSpecificTest/NH2201/Fixture.cs

```diff
@@ -54,5 +55,25 @@ public void CanUseMutliCriteriaAndFetchSelect()
 				Console.WriteLine("*** end");
 			}
 		}
+
+		[Test]
+		public void CanUseQueryBatchAndFetchSelect()
+		{
+			using (var s = OpenSession())
+			{
+				Console.WriteLine("*** start");
```

Comment:
I think it's better to remove these "*** star*" and "*** end" logging in ported tests.